editor.planx.uk/src/routes/flow.tsx

```diff
@@ -174,98 +176,83 @@ const nodeRoutes = mount({
   "/:parent/nodes/:id/edit": editNode,
 });
 
-interface FlowMetadata {
-  flowSettings: FlowSettings;
-  flowAnalyticsLink: string;
-  isFlowPublished: boolean;
-}
-
-interface GetFlowMetadata {
-  flows: {
-    flowSettings: FlowSettings;
-    flowAnalyticsLink: string;
-    publishedFlowsAggregate: {
-      aggregate: {
-        count: number;
-      };
-    };
-  }[];
-}
-
-const getFlowMetadata = async (
-  flowSlug: string,
-  team: string,
-): Promise<FlowMetadata> => {
-  const {
-    data: { flows },
-  } = await client.query<GetFlowMetadata>({
-    query: gql`
-      query GetFlow($slug: String!, $team_slug: String!) {
-        flows(
-          limit: 1
-          where: { slug: { _eq: $slug }, team: { slug: { _eq: $team_slug } } }
-        ) {
-          id
-          flowSettings: settings
-          flowAnalyticsLink: analytics_link
-          publishedFlowsAggregate: published_flows_aggregate {
-            aggregate {
-              count
-            }
-          }
-        }
-      }
-    `,
-    variables: {
-      slug: flowSlug,
-      team_slug: team,
-    },
-  });
-
-  const flow = flows[0];
-  if (!flows) throw new NotFoundError(`Flow ${flowSlug} not found for ${team}`);
-
-  const metadata = {
-    flowSettings: flow.flowSettings,
-    flowAnalyticsLink: flow.flowAnalyticsLink,
-    isFlowPublished: flow.publishedFlowsAggregate?.aggregate.count > 0,
-  };
-  return metadata;
-};
+const SettingsContainer = () => (
+  <Box sx={{ width: "100%", p: 2, overflowY: "auto" }}>
+    <View />
+  </Box>
+)
 
 const routes = compose(
   withData((req) => ({
     flow: req.params.flow.split(",")[0],
   })),
 
-  withView(async (req) => {
-    const [flow, ...breadcrumbs] = req.params.flow.split(",");
-    const { flowSettings, flowAnalyticsLink, isFlowPublished } =
-      await getFlowMetadata(flow, req.params.team);
-    useStore.setState({ flowSettings, flowAnalyticsLink, isFlowPublished });
-
-    return (
-      <>
-        <ErrorBoundary FallbackComponent={ErrorFallback}>
-          <TestEnvironmentBanner />
-          <FlowEditor key={flow} flow={flow} breadcrumbs={breadcrumbs} />
-        </ErrorBoundary>
-        <View />
-      </>
-    );
-  }),
+  withView(flowEditorView),
 
   mount({
     "/": route(async (req) => {
       return {
         title: makeTitle([req.params.team, req.params.flow].join("/")),
-        view: <span />,
-      };
+        view: () => {
+          const [flow, ...breadcrumbs] = req.params.flow.split(",");
+          return (
+            <FlowEditor key={flow} flow={flow} breadcrumbs={breadcrumbs} />
+          )
+        }
+      }
     }),
 
-    "/nodes": nodeRoutes,
+    "/nodes": compose(
+      withView((req) => {
+        const [flow, ...breadcrumbs] = req.params.flow.split(",");
+        return (
+          <>
+            <FlowEditor key={flow} flow={flow} breadcrumbs={breadcrumbs} />
```

Comment:
This duplication of `FlowEditor` is worth being aware of. It matches the current setup where this is attached the view and duplicated that way (just a `<span/>` is rendered in the current setup.

It's the cause of our re-rendering when we go to `/node` routes - react-navi is re-rendering all other views, even those unaffected for some reason.

I repeated previous attempts with memoization here to attempt to solve this with the same results as before. 

The `Header` is also re-rendering when children change, so this may be a wider (or higher level) issue. Happy to carry on looking into it without holding up this PR.